maa.jwt.verifier.sevsnp/readme.md

```diff
@@ -0,0 +1,104 @@
+# SEV-SNP Attestation Sample with Azure Attestation
+
+This is a minimal .NET console application that interacts with the Microsoft Azure Attestation (MAA) service for SEV-SNP attestation type.
+
+## Remote Attestation Flow with SEV-SNP
```

Comment:
I suggest we remove this whole section -- it's distracting and not the area we're trying to explain.

Reply:
removed